docs/imviz/import_data.rst

```diff
@@ -0,0 +1,34 @@
+.. _imviz-import-data:
+
+***********
+Import Data
+***********
+
+Imviz can load data in the form of a filename (string), .jpg, .png,
+an astropy object, or a numpy array if the data is 2D.
+
+Importing data through the GUI
+------------------------------
+
+The first way that users can load their data into the Imviz application is
```

Comment:
In `displaying_images` you addressed the reader as "you", but here you're talking about "the user" in the abstract. I think you should use the former here for consistency.